js/cards.js

```diff
@@ -79,6 +84,12 @@ export async function defineCardComponent() {
       // remove the previous hover effects only if the gacha is not displayed as a grid
       if (gacha_display_selection !== "gacha-grid") {
         this.addEventListener("animationend", addUnclickableToCardsExceptLast);
+        document.getElementById("pile-pull-announcement").textContent =
+          this.getSubtitleText();
+      } else {
+        this.subtitle.classList.remove("hidden");
+        // Set aria-live on parent container to announce subtitle text when it appears
+        this.setAttribute("aria-live", "polite");
```

Comment:
todo: since I set aria-hidden on the subtitle, this live announcement doesn't work anymore
I'll probably have pile-pull-announcement in both gacha views and have it be visually hidden in grid view

Reply:
done